src/sandbox/functionsSimulations/index.ts

```diff
@@ -0,0 +1,93 @@
+import * as functionsToolkit from "@chainlink/functions-toolkit";
+import { DecodedResult } from "@chainlink/functions-toolkit/dist/decodeResult";
+import { ReturnType } from "@chainlink/functions-toolkit/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { DEFAULT_PORT } from "../../shared/constants";
+
+export const simulateRequest = async (
+  hre: HardhatRuntimeEnvironment,
+  source: string,
+  args?: string[],
+  bytesArgs?: string[]
+): Promise<DecodedResult> => {
+  const simulationDeployment =
+    await functionsToolkit.startLocalFunctionsTestnet(
+      undefined,
+      {},
+      hre.config.chainlink.functions_simulation?.port || DEFAULT_PORT
+    );
+
+  const provider = new hre.ethers.providers.JsonRpcProvider(
+    `http://localhost:${
+      hre.config.chainlink.functions_simulation?.port || DEFAULT_PORT
+    }/`
+  );
+  const admin = new hre.ethers.Wallet(
+    simulationDeployment.adminWallet.privateKey,
+    provider
+  );
+  const functionsConsumerAddress =
+    await hre.chainlink.sandbox.functionsConsumer.deploy(
+      simulationDeployment.functionsRouterContract.address,
```

Comment:
line 31 deploys the functions consumer right?  i think `functionsRouterContract` collides with the actual router contracts -- the one in the registry files.  this should be `functionsConsumerContract` insteead?